source/material_model/rheology/elasticity.cc

```diff
@@ -301,6 +323,13 @@ namespace aspect
             for (unsigned int i=0; i < in.n_evaluation_points(); ++i)
               {
                 // Get old stresses from compositional fields
+                //
+                // TODO: Also use the stresses from the old_solution, not just the strain rates.
+                // Explanantion: When this function is called for the assembly of the advection
+                // equations for the stress components, in.composition contains the
+                // values of the current_linearization_point, which is extrapolated
+                // from the old_solution and the old_old_solution.
```

Comment:
Can you remind us what the old_solution and old_old_solution are?

Reply:
These are the solutions for the governing variables obtained in the last and second to last timestep.

Reply:
Thank you :)